ts/who_likes_it.ts

```diff
@@ -0,0 +1,19 @@
+export function likes(a: string[]): string {
+  if (a.length === 0) {
```

Comment:
When you see many if conditions - it is a good practice to try Switch condition.
see this https://www.tutorialsteacher.com/typescript/typescript-switch

Reply:
I think there were 1 more to solve, right? Will you make 1 more PR?